pwa/src/components/dashboardCard/DashboardCard.tsx

```diff
@@ -34,6 +43,10 @@ export const DashboardCard: React.FC<DashboardCardProps> = ({ title, type, statu
         </Link>
       </div>
 
+      <div>
+        <button onClick={deleteOnClick}>Remove</button>
```

Comment:
I think we should use the <Button /> component from @gemeente/denhaag here, instead of a standard HTML one, like so:

```
<Button className={styles.buttonIcon} onClick={onDelete}>
          <FontAwesomeIcon icon={faTrash} />
          {t("Remove")}
        </Button>
```

Note: make sure to mimic the styling from other delete buttons, e.g. from `FormHeaderTemplate`.